pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -179,8 +218,13 @@ func (r *NetworkInfoReconciler) Start(mgr ctrl.Manager) error {
 	return nil
 }
 
-// GarbageCollector collect vpc which has been removed from networkInfo crd.
-// cancel is used to break the loop during UT
+// The GC logic for nsx-vpc is that:
```

Comment:
Comment should have the following format 'GarbageCollector ...' (with an optional leading article)